chainer/functions/connection/linear.py

```diff
@@ -58,17 +58,34 @@ def linear(x, W, b=None):
 
     It accepts two or three arguments: an input minibatch ``x``, a weight
     matrix ``W``, and optionally a bias vector ``b``. It computes
-    :math:`Y = xW^\\top + b`.
+     .. math:: Y = xW^\\top + b.
+
+    .. admonition:: Example
+
+        >>> x = np.random.uniform(0, 1, (3, 4)).astype('f')
+        >>> W = np.random.uniform(0, 1, (5, 4)).astype('f')
+        >>> b = np.random.uniform(0, 1, (5,)).astype('f')
+        >>> y = F.linear(x, W, b)
+        >>> y.shape
+        (3, 5)
 
     Args:
-        x (~chainer.Variable): Input variable. Its first dimension is assumed
-            to be the *minibatch dimension*. The other dimensions are treated
-            as concatenated one dimension whose size must be ``N``.
-        W (~chainer.Variable): Weight variable of shape ``(M, N)``.
-        b (~chainer.Variable): Bias variable (optional) of shape ``(M,)``.
+        x (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`): Input variable, which is :math:`(b, s_1, \
```

Comment:
Could you change the notation to indicate the first dimension of the input variable? Because `b` is already used as the name for the bias, an input argument, so it might be a little confusing for someone to see the definitions of linear link at the first time.